spring-boot-starter/starter-security/src/main/java/org/camunda/bpm/spring/boot/starter/security/oauth2/impl/AuthorizeTokenFilter.java

```diff
@@ -54,6 +56,7 @@ public class AuthorizeTokenFilter extends OncePerRequestFilter {
 
   private static final Logger logger = LoggerFactory.getLogger(AuthorizeTokenFilter.class);
   private final OAuth2AuthorizedClientManager clientManager;
+  private final Clock clock = Clock.systemUTC();
```

Comment:
❓ We usually use `ClockUtil.getCurrentTime()`, should we stick to it? Or what's the advantage of `Clock.systemUTC()`?

Reply:
I took it from the Spring Security classes: [ref](https://github.com/spring-projects/spring-security/blob/main/oauth2/oauth2-client/src/main/java/org/springframework/security/oauth2/client/RefreshTokenOAuth2AuthorizedClientProvider.java#L52).
I'll check if it's an easy update to `ClockUtil` tomorrow.

Reply:
Updated to ClockUtil.